### ISSUE TITLE
Mapbox API free account exceeded the limit

### ISSUE THREAD

**Issue by alicoding** [mozilla/learning.mozilla.org#2368]
Apparently the default account that I've set on this repo for testing purposed was used on production...

https://github.com/mozilla/learning.mozilla.org/blob/8624e0c899a211bd75f6976081b64d9215532152/components/map.jsx#L16

And it went over the limit for free account and Mapbox wanted to charge my account $23 for went over the limi (kinda stupid because it should warn me first before doing so).

**Comment by alicoding** [comment]
This is being used here https://learning.mozilla.org/en-US/clubs @gideonthomas @cadecairos @hannahkane ^^

**Comment by cadecairos** [comment]
I'll handle this today

**Comment by cadecairos** [comment]
I've taken control of ali's account (which he doesn't need anymore) so that we can pay for the usage.

I also created a new account in mapbox and set up the site to use a new map on that account and an access token associated with that account.

As for subscribing - I don't think it's cost effective at this point in time to subscribe. The next plan up from free is a mind-boggling $500/month. With a non-profit discount applied, we're still looking at $450/month.

Based on the overage charges we recieved, we can keep the free plan and still pay less, even up to about 800k monthly page views on `/clubs`

In any case, I think we can consider this closed.
